lib/dau_web/live/view_external_escalations_live.ex

```diff
@@ -0,0 +1,73 @@
+defmodule DAUWeb.ViewExternalEscalationsLive do
+  use DAUWeb, :live_view
+
+  alias DAU.ExternalEscalation
+  import DAUWeb.CoreComponents
+
+  @impl true
+  def mount(_params, _session, socket) do
+    case ExternalEscalation.list_form_entries() do
+      {:error, _reason} ->
+        {:ok,
+         socket
+         |> assign(entries: [])
+         |> put_flash(:error, "Could not load escalation entries. Please try again later.")}
+      entries ->
+        sorted_entries = Enum.sort_by(entries, & &1.inserted_at, {:desc, DateTime})
```

Comment:
this sorting should happen at the database level. 
you can modify `list_form_entries` to query Postgres for results sorted by inserted_at